plugins/github-actions/src/components/useWorkflowRuns.ts

```diff
@@ -41,12 +42,12 @@ export function useWorkflowRuns({
   owner,
   repo,
   branch,
-  initialPageSize = 5,
+  initialPageSize = 6,
```

Comment:
More curious then anything but why is this change needed?

Reply:
Card view UI was looking good when it have 6 cards in one page than 5 to avoid empty space, so I changed to 6.